### PR TITLE
Pagination api am

### PR DESCRIPTION
**Pull request by Arcziwald**
I connected the API to the previously styled Pagination.

### CODE REVIEW

File: src/common/Pagination/styled.js [hunk 1]
```diff
@@ -24,74 +21,63 @@ export const Button = styled.button`
...
   transition: 0.5s;
   cursor: pointer;
-
-  &:hover {
+  &:hover:not(:disabled) {
     filter: brightness(103%);
   }
   &:disabled {
     background-color: ${({ theme }) => theme.colors.mystic};
+    cursor: not-allowed;
```
Comment (KrystianZybura) [inline src/common/Pagination/styled.js]: I think it might be a little too scary for the user, default cursor will do just fine 👌

File: src/common/Pagination/styled.js
Comment (KrystianZybura) [inline src/common/Pagination/styled.js:1]: I'm not sure why did you remove all the code spaces, now it might be harder to read.. 😅

File: src/common/Pagination/index.js
Comment (KrystianZybura) [inline src/common/Pagination/index.js:1]: Similar thing with code spaces.. ❔

File: src/common/PeopleList/index.js
Comment (KrystianZybura) [inline src/common/PeopleList/index.js:1]: Great job here 💪

File: src/features/MoviesBrowser/PopularPeople/usePopularPeopleData.js
Comment (KrystianZybura) [inline src/features/MoviesBrowser/PopularPeople/usePopularPeopleData.js:1]: Amazing code refactoring and implementation 💪

**Review by KrystianZybura — COMMENTED** [review]
Amazing work, just few minor things 😎

Also, this should be corrected 😅
[
image
](url)

File: src/common/Pagination/styled.js (hunk 1 above)
Comment (Arcziwald) [reply src/common/Pagination/styled.js]: done ;-)

File: src/common/Pagination/styled.js
Comment (Arcziwald) [reply src/common/Pagination/styled.js:1]: done

File: src/common/Pagination/index.js
Comment (Arcziwald) [reply src/common/Pagination/index.js:1]: done

**Comment by Arcziwald** [comment]
> Amazing work, just few minor things 😎
> 
> Also, this should be corrected 😅 [ ](url)image

no way works to get 24 items. API only provides 20... more details for later ;-)

**Comment by KrystianZybura** [comment]
> > Amazing work, just few minor things 😎
> > Also, this should be corrected 😅 [ ](url)image
> 
> no way works to get 24 items. API only provides 20... more details for later ;-)

The thing is, there is a double pagination here, I don't really mind the photos, since they're not related to pagination. Apologies, I could've been more precise with it..

**Comment by Arcziwald** [comment]
> > > Amazing work, just few minor things 😎
> > > Also, this should be corrected 😅 [ ](url)image
> > 
> > 
> > no way works to get 24 items. API only provides 20... more details for later ;-)
> 
> The thing is, there is a double pagination here, I don't really mind the photos, since they're not related to pagination. Apologies, I could've been more precise with it..

cool :-) I'm removing inactive pagination :-)

**Review by KrystianZybura — APPROVED** [review]

**Review by WoytekMig — APPROVED** [review]
Everything seems working very well :) Good job ! :)
